requirements.txt

```diff
@@ -9,4 +9,4 @@ pandas==1.1.4
 PyYAML==5.4
 requests==2.25.0
 tqdm==4.52.0
-xlrd==1.2.0
```

Comment:
Can we leave this dependency in just in case some of the other tests (for cocoon) that read excel files?

Reply:
Sure.